keywording/text.xml

```diff
@@ -237,23 +242,57 @@ to do this.
 <body>
 
 <p>
-Stabilization, i.e., moving an ebuild from <c>~arch</c> to <c>arch</c>, is done
-by the relevant architecture teams. If you have access to exotic hardware but
-are not on the arch teams, you may wish to make individual arrangements <d/>
-the arch teams are happy for help, so long as they know what is going on.
+Stabilization, i.e., moving an ebuild from <c>~arch</c> ("testing") to
+<c>arch</c> ("stable"), is done by the relevant architecture teams. If you have
+access to exotic hardware but are not on the arch teams, you may wish to make
+individual arrangements <d/> the arch teams are happy for help, so long as they
+know what is going on.
 </p>
 
 <p>
 In order to request stabilization of an ebuild, file a bug to the package's
 maintainer (which may be yourself) in the Stabilization component, and list
 all secondary maintainers in the bug's CC. When the maintainers consider the
 ebuild to be ready for stabilization, they will add the relevant architecture
-teams to the CC list. They can do it manually, or they can fill the package
+teams to the CC list. They can fill the package
 list field, add the <c>CC-ARCHES</c> keyword, and let
 <uri link="https://dev.gentoo.org/~mgorny/doc/nattka/">NATTkA</uri>
-automatically add arch teams to CC.
-That way teams can remove themselves from the list when they are done, giving
-a clear indication of which teams still have to stabilize a package.
+automatically add arch teams to CC (preferred), or do this manually.
+That way, teams can remove themselves from the list when they are done, giving
+a clear indication of which teams still remain to stabilize a package.
+</p>
+
+<p>
+It is <b>strongly</b>
```

Comment:
Normal emphasis in running text should be done with `<e>`, not `<b>`.

(`<b>` is for things like important keywords that should catch the eye, but not for words like "not", "only", or "strongly".)

Reply:
Got it.